app/Tasks/Stop.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Tasks;
+
+use App\Builder\Docker;
+use App\Builder\DockerCompose;
+
+class Stop extends Task
+{
+    /** @var bool $purge */
+    protected $purge = true;
+
+    public function run(...$args): int
+    {
+        $tasks = [
+            [$this, 'destroyContainers'],
+            [$this, 'handleNetwork'],
+        ];
+
+        return $this->runCallables($tasks);
+    }
+
+    public function purge(bool $purge) : self
+    {
+        $this->purge = $purge;
+
+        return $this;
+    }
+
+    public function handleNetwork() : int
+    {
+        return $this->runTask('Destroy network', function () {
+            return $this->runCommandWithoutOutput(
+                Docker::makeWithDefaultArgs('network', 'rm', env('FWD_NETWORK'))
```

Comment:
maybe only destroy global if we call purge?

Reply:
well, while networks are ephemeral, the global network in the future could be used by other running projects, so indeed removing it is not so simple. Let's think this through.

Reply:
better remove for now